### ISSUE TITLE
ActivateViewer button may not show up when two Project tabs are open in the same viewport and the first one is focused

### ISSUE THREAD

**Issue by hai-vr** [hai-vr/animation-viewer-vcc#1]
See https://github.com/hai-vr/blendshape-viewer/issues/8

**Comment by mekanyanko** [comment]
Hi,
I wanted to let you know that I was able to fix the issue I encountered. 

Solution:
Ensured that every GUILayout.BeginScrollView call has a corresponding GUILayout.EndScrollView call.
Added null checks for reflection calls to avoid TargetException.

Everything is working correctly now. Thank you for your hard work on this tool!

DiffReport:
[AnimationViewerEditorWindow_DiffReport.zip](https://github.com/user-attachments/files/16324435/AnimationViewerEditorWindow_DiffReport.zip)
